README.md

```diff
@@ -8,62 +11,83 @@
+* Website: https://coraza.io
+* OWASP Slack Community (#coraza): https://owasp.org/slack/invite
+* Issue Tracker: https://github.com/corazawaf/coraza/issues
+* Coraza Playground: https://playground.coraza.io
+* Project Planning: https://github.com/orgs/corazawaf/projects?type=beta
 
-## Prerequisites
+Coraza is an open source, enterprise-grade, high performance Web Application
+Firewall (WAF) ready to protect your beloved applications. It written in Go,
+supports ModSecurity SecLang rulesets and is 100% compatible with the OWASP
+Core Rule Set.
 
-* Linux distribution (Debian and Centos are recommended, Windows is not supported yet)
-* Golang compiler v1.16+
+* **Firewall Engine** Coraza is an implementation of the SecLang engine in the
+    memory-safe Go language. Coraza runs the [OWASP Core Rule Set
+    (CRS)](https://coreruleset.org/) to stop attacks and generate important audit
+    information.
 
+* **Security** - Coraza runs the [OWASP Core Rule Set (CRS)](https://coreruleset.org)
+		which protects web applications from a wide range of attacks, including the
+		OWASP Top Ten, with a minimum of false alerts. CRS protects from many
+		common attack categories including: SQL Injection (SQLi), Cross Site
+		Scripting (XSS), Local File Inclusion (LFI), Remote File Inclusion (RFI),
+		PHP Code Injection, Java Code Injection, HTTPoxy, Shellshock, Unix/Windows
+		Shell Injection, Session Fixation, Scripting/Scanner/Bot Detection,
+		Metadata & Error Leakages.
 
-## Migrate from v1
+* **Performance** - From huge websites to small blogs, Coraza can handle that load
+    with minimal performance impacts. Check our [Benchmarks](https://coraza.io/docs/reference/benchmarks)
 
-* Rollback SecAuditLog to the legacy syntax (serial/concurrent)
-* Attach an error log handler using ```waf.SetErrorLogCb(cb)``` (optional)
-* the function Transaction.Clean() must be used to clear transaction data, files and take them back to the sync pool.
-* If you are using low level APIs check the complete changelog as most of them were removed.
-* OWASP CRS does not require any external dependency anymore
+* **Integrated** - Coraza is a library at its core, but we support many
+    integrations to deploy a WAF as an application server, reverse proxy,
+    container, and more.
 
+* **Extensible** - Audit Loggers, persistence engines, operators, actions,
+    create your own functionalities to extend Coraza as much as you want.
 
-## Running the tests
 
-Run the go tests:
+## Implementations
 
-```sh
-go test ./...
-go test -race ./...
-```
+The Coraza Project maintains implementations and plugins for the following servers: 
 
-Using pre-commit 
+* [Caddy Reverse Proxy and Webserver Plugin](https://github.com/corazawaf/coraza-caddy) (stable, needs a maintainer)
+* [HAProxy SPOE Plugin](https://github.com/jptosso/coraza-spoa) (preview)
```

Comment:
Are these links good? They point to your personal account.

Reply:
Indeed, please fix links